### DISCUSSION TITLE
Is it necessary to implement additional features in the example scenario "Kinematic orbit determination of LEO satellites" ?

### DISCUSSION THREAD

**Discussion by kai-ljk** [groops-devs/groops#257] [Ideas]
Dear developers,

In the example scenario "Kinematic orbit determination of LEO satellites",
I noticed that an "approximate orbit" is required to conduct the GnssProcessing. 
While no such an orbit available for a totally new stellite mission.

Might it be necessary  to implement Single Point Poisitiong and Dynamit fitting functions, or just like to generate an approximate orbit
  in the scenario?

**Comment by kai-ljk** [comment]
Another  alternative may be using GnssReceiverGeneratorStationNetwork() to process the LEO as a StationNetwork with only one station in the network. After that,  coarse positions will be output, and can be used by PreprocessingVariationalEquation to get an smoother orbit.
I am going to try on that.

**Comment by AndreHauschild** [comment]
You could use TLEs for a first approximation of the orbit.

**Reply by Hanbing59** [reply]
Maybe for some satellites or at the very early stage, the TLE is also not available. Would be, for LEOs using SPP by GNSS observations and for new GNSS satellites, something like inverse-SPP an option to get the very first coarse orbit for getting following orbit determinations started?
